impls/src/adapters/epicbox.rs

```diff
@@ -840,6 +849,23 @@ impl EpicboxBroker {
 									error!("ProtocolResponse::Error {}", response);
 								}
 							},
+							ProtocolResponseV2::Ok {} => {
+								info!("Subscription Ok.");
+								let duration =
+									std::time::Duration::from_secs(EPICBOX_SUBSCRIPTION_INTERVAL);
+								std::thread::sleep(duration);
```

Comment:
This behavior is fine for now.  But, it looks to me like it is stopping response processing during the sleep. I'd like to handle this more gracefully moving forward. 

But, we can handle that when we don't have a pending necessary release.  ACK for now.